README.md

```diff
@@ -24,25 +24,6 @@ Commands:
   help        Displays help information about the specified command.
 ```
 
-## Download and Installation
```

Comment:
I like this, but at least we should put a pointer (direct link to the correct site in the docs) instead...

Reply:
I've added a wiki tab with a link to our docs, just like in recheck and recheck-web: https://github.com/retest/recheck.cli/wiki.

Reply:
I personally feel that this does not suffice. People are lazy and probably won't find this. I feel this is such an important information, that it should be prominently displayed. Removing it from the readme is already critical, but the link should definitely be there. Also ... you want the repo to be as self-contained as possible, not relying on the github wiki to access a simple link.

Reply:
Then remove the wiki from all repos and simply link to the docs from a "Documentation" section inside the `README.md`?

Reply:
We can do this either way. I just feel that the "how to install that piece of software" should be directly described or at least linked from the readme. So when we remove that info in this PR, we should put a link not to the documentation in general, but to the part where installation is described...

Reply:
OK, added a link.